src/lib/components/dashboard/folders/AddFolder.svelte

```diff
@@ -15,25 +18,33 @@
     folderStore.set(responseJson.data);
     showAddFolderDrawer.set(false);
   };
+
+  const handleClose = () => {
```

Comment:
wouldn't it be a good idea to use store to close rather than dispatching an event?

Reply:
modified